core-addon/background.js

```diff
@@ -2,7 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-const IonCore = require('./IonCore.js');
+var browser = require("../node_modules/webextension-polyfill/dist/browser-polyfill.js");
```

Comment:
There must be some way to not use the full path here, but I failed to find it.

Reply:
`const browser = require("webextension-polyfill");` doesn't work?

Reply:
This seems to work for the test part, but when packing the addon it fails with:

```
$ npm run build-addon

> svelte-app@1.0.0 build-addon /mnt/d/Mozilla/ion/ion-svelte
> rollup -c rollup.config.addon.js


core-addon/background.js → public/addon-build/background.js...
(!) Unresolved dependencies
https://rollupjs.org/guide/en/#warning-treating-module-as-external-dependency
webextension-polyfill (imported by core-addon/background.js, webextension-polyfill?commonjs-external)
created public/addon-build/background.js in 167ms
```

Reply:
Thanks for the help! This is now fixed!